caab-api/open-api-specification.yml

```diff
@@ -679,26 +681,20 @@ components:
           type: 'string'
         award_code:
           type: 'string'
-        triggering_event:
```

Comment:
Were these 3 attributes not needed? They are in Award.java in the PUI.

Reply:
Please see: https://dsdmoj.atlassian.net/browse/CCLS-1932

Reply:
Ha ha, brilliant. The bug that I raised.